howso/prediction_stats.amlg

```diff
@@ -484,7 +489,14 @@
 					mode (if (= robust_hyperparameters (true)) "robust" "full")
 					weight_feature (if (= weight_feature (null)) ".none" weight_feature)
 				))
-			features (if (!= action_feature (null)) (append context_features action_feature) context_features)
+			features 
```

Comment:
I figure the declaration of `context_features` above is what should be altered in this change. It seems that context features could still be incorrect in flows where `prediction_stats_features` is specified.